thunder/examine/__init__.py

```diff
@@ -272,3 +274,59 @@ def get_nvfuser_repro(trace: TraceCtx, fusion_name: str, /) -> str:
         raise RuntimeError("The installed version of nvFuser does not support repro generation unless on crash.")
 
     return get_repro(fusion.last_inputs)
+
+
+def make_trace_dot(trace: TraceCtx):
+    """
+    Creates a directed graph of the given trace.
+
+    This function is intended to be used to use graphviz to visualize the computation graph of a trace.
+    Beware, rendering out a graph for large traces might take a while.
+
+    Requires graphviz to be installed, for more information check out -> https://graphviz.readthedocs.io/en/stable/index.html
+
+    Args:
+        trace (TraceCtx): The Thunder trace to be made into a graph.
+
+    Returns:
+        graphviz.Digraph: A graphviz directed graph.
+    """
+    if not importlib.util.find_spec("graphviz"):
+        warn("graphviz is not available. Graph cannot be created.")
+        return
+
+    import graphviz
```

Comment:
PyTorch FX uses `pydot`. What's the difference from `graphviz` and should one be preferred over another?
https://github.com/pytorch/pytorch/blob/7f88bf96f9ff37f3873cc5cb8c962ebcbf509c65/torch/fx/passes/graph_drawer.py#L112